SubZero/src/model/filter/Filter.cpp

```diff
@@ -16,3 +16,6 @@ Filter::~Filter() {
 	// TODO Auto-generated destructor stub
 }
 
+char getCreator(){
```

Comment:
Oh, originally I wanted to track who created the filter as an added info. But after the meeting I decided to take it out, this must have escaped.